README.md

```diff
@@ -71,30 +72,51 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
+| :white_check_mark:  [DFF](configs/vid/dff) (CVPR 2017)                              | :white_check_mark: [ILSVRC](http://image-net.org/challenges/LSVRC/2017/) |
+| :white_check_mark: [FGFA](configs/vid/fgfa) (ICCV 2017)                             |                                                                          |
+| :white_check_mark: [SELSA](configs/vid/selsa) (ICCV 2019)                           |                                                                          |
+| :white_check_mark: [Temporal RoI Align](configs/vid/temporal_roi_align) (AAAI 2021) |                                                                          |
+
+### Single Object Tracking
+
+| Method                                                                 | Dataset                                                                 |
+| ---------------------------------------------------------------------- | ----------------------------------------------------------------------- |
+| :white_check_mark: [SiameseRPN++](configs/sot/siamese_rpn) (CVPR 2019) | :white_check_mark:    [LaSOT](http://vision.cs.stonybrook.edu/~lasot/)  |
+| :white_check_mark: [STARK](configs/sot/stark) (ICCV 2021)              | :white_check_mark:    [UAV123](https://cemse.kaust.edu.sa/ivul/uav123/) |
+|                                                                        | :white_check_mark:  [TrackingNet](https://tracking-net.org/)            |
+|                                                                        | :white_check_mark:  [OTB100](http://www.visual-tracking.net/)           |
+|                                                                        | :white_check_mark:  [GOT10k](http://got-10k.aitestunion.com/)           |
+|                                                                        | :white_check_mark:  [VOT2018](https://www.votchallenge.net/vot2018/)    |
+
+### Multi-Object Tracking
+
+| Method                                                                     | Dataset                                                         |
+| -------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| :white_check_mark:  [SORT/DeepSORT](configs/mot/deepsort) (ICIP 2016/2017) | :white_check_mark:  [MOT Challenge](https://motchallenge.net/)  |
+| :white_check_mark:  [Tracktor](configs/mot/tracktor) (ICCV 2019)           | :white_check_mark:    [CrowdHuman](https://www.crowdhuman.org/) |
+| :white_check_mark:  [QDTrack](configs/mot/qdtrack) (CVPR 2021)             | :white_check_mark:   [LVIS](https://www.lvisdataset.org/)       |
+| :white_check_mark:  [ByteTrack](configs/mot/bytetrack) (arXiv 2021)        | :white_check_mark:   [TAO](https://taodataset.org/)             |
+| :o: [OC-SORT](https://github.com/noahcao/OC_SORT)  (arXiv 2022)            | :o: [DanceTrack](https://dancetrack.github.io)                  |
```

Comment:
It is unnecessary to set the link for on the way methods

Reply:
I great. Will edit them.